src/faebryk/libs/picker/api/common.py

```diff
@@ -271,9 +271,7 @@ def check_compatible_parameters(
 
     # check for every param whether the candidate component's range is
     # compatible by querying the solver
-    anded = And(
-        *(m_param.operation_is_superset(c_range) for m_param, c_range in param_mapping)
```

Comment:
if we pick a cap for X.capacitance, nobody is allowed to constrain_superset with a wider set than the part has so superset is not correct.